third_party/move/tools/move-smith/src/config.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+pub struct Config {
+    pub max_num_modules: usize,
+    pub max_num_functions_in_module: usize,
+    pub max_num_structs_in_module: usize,
+    pub max_num_uses_in_module: usize,
+    pub max_num_friends_in_module: usize,
+    pub max_num_constants_in_module: usize,
+    pub max_num_specs_in_module: usize,
+
+    pub max_num_fields_in_struct: usize,
+
+    pub max_num_stmt_in_func: usize,
+    pub max_num_params_in_func: usize,
+
+    pub max_num_calls_in_script: usize,
+}
+
+impl Default for Config {
```

Comment:
What criteria were used to pick those numbers?

Reply:
They were just randomly chosen for now.